src/components/Editor/EditorToolbarButton.module.css

```diff
@@ -7,3 +7,16 @@
 .toolbarButton.smallPadding {
   padding-inline: 8px;
 }
+
+.toolbarButton,
+.toolbarButton:hover {
+  color: #4a4a4a;
```

Comment:
In the future these can be variables so they will be customized

Reply:
I agree 💪🏼, we're not there yet due to time constraints